VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -70,14 +76,31 @@ public void getItemOffsets(Rect outRect, View view, RecyclerView parent, Recycle
         });
         discussionCommentsListView.setAdapter(discussionCommentsAdapter);
 
-        createNewCommentTextView.setText(context.getString(R.string.discussion_post_create_new_comment));
-
-        createNewCommentLayout.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                router.showCourseDiscussionAddComment(context, discussionComment);
-            }
-        });
+        if (isTopicClosed) {
+            createNewCommentTextView.setText(context.getString(R.string.discussion_add_comment_disabled_title));
+            TextViewCompat.setCompoundDrawablesRelative(createNewCommentTextView,
+                new IconDrawable(getActivity(), Iconify.IconValue.fa_lock)
+                        .sizeRes(R.dimen.icon_view_standard_width_height)
+                        .colorRes(R.color.edx_grayscale_neutral_white_t),
+                null, null, null
+            );
+            createNewCommentLayout.setEnabled(false);
+        }
+        else {
+            createNewCommentTextView.setText(context.getString(R.string.discussion_post_create_new_comment));
+            TextViewCompat.setCompoundDrawablesRelative(createNewCommentTextView,
+                    new IconDrawable(getActivity(), Iconify.IconValue.fa_plus_circle)
+                            .sizeRes(R.dimen.icon_view_standard_width_height)
+                            .colorRes(R.color.edx_grayscale_neutral_white_t),
+                    null, null, null
+            );
+            createNewCommentLayout.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    router.showCourseDiscussionAddComment(context, discussionComment);
+                }
+            });
```

Comment:
Since this whole setup logic is being used in multiple Fragments, maybe we should have a utility method for it as well (`DicussionUtils`?), taking the texts and a click listener as parameters.